sesam.py

```diff
@@ -687,7 +686,6 @@ def parse_config_file(self, filename):
 
         return config
 
-
     def read_config_file(self, filename, is_required=False):
```

Comment:
This method includes the main changes.
The rest of the changes are due to PyCharm reformatting.